src/cli/bin.ts

```diff
@@ -36,36 +69,112 @@ yargs
         demandOption: false,
         desc: "The path to the JSON export of the /pivot folder on the content server.",
       });
+      args.option("debug", {
+        type: "boolean",
+        demandOption: false,
+        default: false,
+        desc: "Whether to create a file containing details about the errors encountered during the migration.",
+      });
+      args.option("stack", {
+        type: "boolean",
+        demandOption: false,
+        default: false,
+        desc: "Whether to include stacktraces in the error report file.",
+      });
     },
     async ({
       inputPath,
       outputPath,
       serversPath,
       removeWidgets: keysOfWidgetPluginsToRemove,
       pivotInputPath,
+      debug,
+      stack,
     }: {
       inputPath: string;
       outputPath: string;
       serversPath: string;
       removeWidgets: string[];
       pivotInputPath?: string;
+      debug: boolean;
+      stack: boolean;
     }) => {
       const legacyUIFolder = await fs.readJSON(inputPath);
       const legacyPivotFolder = pivotInputPath
         ? await fs.readJSON(pivotInputPath)
         : undefined;
       const servers = await fs.readJSON(serversPath);
 
-      const migratedUIFolder = await migrateUIFolder(
+      const [migratedUIFolder, counters, errorReport] = await migrateUIFolder(
         legacyUIFolder,
-        servers,
-        keysOfWidgetPluginsToRemove,
-        legacyPivotFolder,
+        {
+          legacyPivotFolder,
+          servers,
+          keysOfWidgetPluginsToRemove,
+          doesReportIncludeStacks: stack,
+        },
       );
 
+      const { dir } = path.parse(outputPath);
+
       await fs.writeJSON(outputPath, migratedUIFolder, {
         spaces: 2,
       });
+
+      console.log("--------- END OF CONTENT MIGRATION ---------");
+
+      Object.entries(counters)
+        .filter(([, countersForFolder]) =>
+          Object.values(countersForFolder).some((value) => value > 0),
+        )
+        .forEach(([folderName, countersForFolder]) => {
+          console.log(`\n# ${_capitalize(folderName)}`);
+          Object.entries(countersForFolder).forEach(([outcome, counter]) => {
+            if (counter > 0) {
+              console.log(
+                `- ${counter} ${summaryMessages[folderName][outcome]}`,
+              );
+            }
+          });
+        });
+
+      if (
+        counters.dashboards.failed +
+          counters.dashboards.partial +
+          counters.filters.failed +
+          counters.widgets.failed >
+        0
+      ) {
+        console.log("\nRemarks:");
+        console.log(
+          "- the `text-editor` widget is not part of the plugin registry in the ActiveUI 5 starter. Make sure to add it in your project, see https://activeviam.com/activeui/documentation/latest/docs/tutorial/your-first-custom-widget#extend-activeui.",
```

Comment:
As mentioned above, I would prefer to not bother people who do not have a single `text-editor` widget in their content with this information.